File: python/easy/construct_the_rectangle.py
class Solution(object):
    def constructRectangle(self, area):
        """
        :type area: int
        :rtype: List[int]
        """
        sqrt = int(math.sqrt(area))
        while area % sqrt != 0:
            sqrt -= 1
        return [area/sqrt, sqrt]

File: c++/easy/construct_the_rectangle.cpp
class Solution {
public:
    vector<int> constructRectangle(int area) {
        vector<int> rect;
        if(area <= 0)
            return rect;
        int r = 1;
        for(int i = 1; i*i <= area; ++i)
        {
            if(area % i == 0)
                r = i;
        }
        rect.push_back(area/r);
        rect.push_back(r);
        return rect;
    }
};

Conclustion:
easy! math algorithm issue
